### PR TITLE
fix(Environment/Deployments): remove environments and deployments references from UI

### PR DESCRIPTION
**Pull request by sahil143**
## Fixes 

https://issues.redhat.com/browse/HAC-5682

## Description

Removed Environments from:
- Side Navigation
- Environment list page
- Create environment form
- App workflow
- Commit workflow
- Add secret form

Removed Deployments from:
- Application details
- Component details
- Snapshot details

## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by wolfeallison** [comment]
Here are the copy changes. Let me know if I missed anything or please leave any comments if any need revisions https://docs.google.com/document/d/1ZT4wfminXxoQGsooWhm9L1FX9wTY9_lhw0qTSLhqRvM/edit?usp=sharing

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by sahil143** [comment]
[test]

**Comment by sahil143** [comment]
[test]

File: src/components/ApplicationDetails/tabs/overview/visualization/hooks/useAppWorkflowData.ts [hunk 1]
```diff
@@ -120,7 +104,6 @@ export const useAppWorkflowData = (
       ...(componentNodes?.length ? componentNodes : [componentGroup]),
       ...(buildNodes?.length ? buildNodes : [buildGroup]),
       ...applicationIntegrationTestNodes,
-      ...(staticEnvironmentNodes?.length ? staticEnvironmentNodes : [staticEnvironmentGroup]),
       ...(mvpFeature ? [] : releaseNodes?.length ? releaseNodes : [releaseGroup]),
```
Comment (rohitkrai03) [inline src/components/ApplicationDetails/tabs/overview/visualization/hooks/useAppWorkflowData.ts:107]: Should we also remove `mvpFeature` flag?

File: src/components/ApplicationDetails/tabs/overview/visualization/hooks/useAppWorkflowData.ts [hunk 2]
```diff
@@ -135,7 +118,6 @@ export const useAppWorkflowData = (
       componentGroup,
       buildGroup,
       testsGroup,
-      staticEnvironmentGroup,
       ...(mvpFeature ? [] : [releaseGroup]),
       ...(mvpFeature ? [] : [managedEnvironmentGroup]),
```
Comment (rohitkrai03) [inline src/components/ApplicationDetails/tabs/overview/visualization/hooks/useAppWorkflowData.ts:122]: I think we need to remove this as well.

File: src/components/IntegrationTest/IntegrationTestForm/IntegrationTestSection.tsx [hunk 3]
```diff
@@ -183,23 +150,13 @@ const IntegrationTestSection: React.FC<React.PropsWithChildren<Props>> = ({ isIn
           data-test="git-path-repo"
           required
         />
-        <DropdownField
```
Comment (rohitkrai03) [inline src/components/IntegrationTest/IntegrationTestForm/IntegrationTestSection.tsx:186]: Do we not support any kind of ephemeral environments for integration tests now?

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]
Lets also get rid of the deprecated files. I don't see a need for keeping them.

**Comment by abhinandan13jan** [comment]
@sahil143 This is failing codecov/project test, thus would also need o add codecov/patch as well

File: src/components/ApplicationDetails/tabs/overview/visualization/hooks/useAppWorkflowData.ts (hunk 2 above)
Comment (sahil143) [reply src/components/ApplicationDetails/tabs/overview/visualization/hooks/useAppWorkflowData.ts:122]: managedEnvironment is ReleasePlan resource.

File: src/components/ApplicationDetails/tabs/overview/visualization/hooks/useAppWorkflowData.ts (hunk 1 above)
Comment (sahil143) [reply src/components/ApplicationDetails/tabs/overview/visualization/hooks/useAppWorkflowData.ts:107]: We can remove this in a seperate PR.

File: src/components/IntegrationTest/IntegrationTestForm/IntegrationTestSection.tsx (hunk 3 above)
Comment (sahil143) [reply src/components/IntegrationTest/IntegrationTestForm/IntegrationTestSection.tsx:186]: As discussed on slack, Right now there is no UI change required in place of Environments for integration tests.

**Review by rohitkrai03 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**